src/displayUI.js

```diff
@@ -71,7 +76,6 @@ export function displayOneLink(link) {
     param_svg,
     param_api,
     param_category,
```

Comment:
please camel case notation, not text_text, notation refactor all variables to be camel case